draft-ietf-scitt-architecture.md

```diff
@@ -1009,6 +1009,16 @@ It is up to the Issuer to notify Transparency Services of credential revocation
 
 # IANA Considerations
 
+## COSE Receipts Header Parameter
```

Comment:
Normally you would request IANA with definitive number ranges for your work!

Please check with @henkbirkholz how best we can handle this ?

Adding IANA Considerations is a great start!

Reply:
This PR is to avoid readers implementing numbers that are not assigned.
While we're pursuing assignment, we should merge this PR to indicate the numbers are not yet assigned.